config/manager/kustomization.yaml

```
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: localhost/tekton-kueue
-  newTag: v2
+  newName: quay.io/konflux-ci/tekton-kueue
```

should we avoid to have a default registry here?

```suggestion
  newName: konflux-ci/tekton-kueue
```